python/rascaline/rascaline/utils/clebsch_gordan/_clebsch_gordan.py

```diff
@@ -6,7 +6,7 @@
 import itertools
 from typing import List, Optional, Tuple, Union
 
-from metatensor import Labels, TensorBlock, TensorMap
+from ._classes  import Labels, TensorBlock, TensorMap
```

Comment:
Should this have been changed? There is a currently a build error because it's trying to import these from rascaline instead of metatensor:
```
File "/home/runner/work/rascaline/rascaline/.tox/torch-tests/lib/python3.8/site-packages/rascaline/utils/clebsch_gordan/correlate_density.py", line 8, in <module>
          from ._classes  import Labels, TensorMap
      ModuleNotFoundError: No module named 'rascaline.utils.clebsch_gordan._classes'
```